script/acceptancetest

```diff
@@ -107,6 +107,11 @@ if [[ $? != 0 ]]; then
   failed=1
 fi
 
+# Shared Filesystems v2
```

Comment:
This is a good idea and I'm actually surprised any tests are passing. It looks like we'll have to review all of the tests and make sure they're using settings suitable for a standard devstack installation. For now, let's omit this.

Reply:
Additionally, feel free to use issue #818 for any PRs that are only for acceptance tests.

Reply:
Seems so. I fixed the tests and now they run fine in my local manila environment, however they still don't run in openlabtesting.org env. I've added a stub in the shell script and created an issue: #1345